.woodpecker/docs.yaml

```
@@ -1,6 +1,6 @@
 variables:
   - &golang_image 'docker.io/golang:1.23'
-  - &node_image 'docker.io/node:23-alpine'
+  - &node_image 'docker.io/node:23.1-alpine'
```

```suggestion
  - &node_image 'docker.io/node:23-alpine'
```

Can be reverted, 23.3. images were published.